src/index.ts

```diff
@@ -22,64 +23,67 @@ const defaultMinifyOptions: HtmlMinifierTerserOptions = {
   minifyCSS: true,
 };
 
-export default function simpleHtmlPlugin({ minify: minifyOptions = false, inject }: Options = {}) {
+export default function simpleHtmlPlugin({ minify: minifyOptions = false, inject, order = "pre" }: Options = {}) {
```

Comment:
I wonder if we even need this option? Is there a good reason to have it? I'd love for this plugin to leave up to its name.